test/integration/requests/multisign.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable mocha/no-hooks-for-single-case -- Makes test setup consistent across files */
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import { AccountSet, Client, SignerListSet, Wallet } from 'xrpl-local'
+import { convertStringToHex } from 'xrpl-local/utils'
+import { multisign, sign } from 'xrpl-local/wallet/signer'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+import {
+  fundAccount,
+  ledgerAccept,
+  testTransaction,
+  verifySubmittedTransaction,
+} from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
```

Comment:
@mvadari Should this TIMEOUT be moved to a constant in `integration/utils`?

Reply:
The whole `this.timeout` could actually be moved into `suiteClientSetup`